frontend/src/components/common/camps/CampSessionInfoHeader.tsx

```diff
@@ -0,0 +1,118 @@
+import { Container, Flex, IconButton, Text, Box } from "@chakra-ui/react";
+import React, { useState } from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faChevronLeft,
+  faChevronRight,
+} from "@fortawesome/free-solid-svg-icons";
+
+import textStyles from "../../../theme/textStyles";
+
+type CampSessionInfoHeaderProps = {
+  camp: any;
+};
+
+const CampSessionInfoHeader = ({
+  camp,
+}: CampSessionInfoHeaderProps): React.ReactElement => {
+  const [currentCampSession, setCurrentCampSession] = useState(0);
+  const numSessions = camp.campSessions.length;
+
+  const nextSession = () => {
+    if (numSessions >= 2)
+      setCurrentCampSession(
+        currentCampSession === numSessions - 1 ? 0 : currentCampSession + 1,
+      );
+  };
+
+  const prevSession = () => {
+    if (numSessions >= 2)
+      setCurrentCampSession(
+        currentCampSession === 0 ? numSessions - 1 : currentCampSession - 1,
```

Comment:
same as above ^

Reply:
actually % in typescript is not a modulo operator but gives the remainder which can be negative :// can just ignore both my comments